cpp/src/quantiles/quantiles.h

```diff
@@ -20,142 +20,13 @@
 
 #include "cudf.h"
 
-#include <thrust/execution_policy.h>
-#include <thrust/system/cuda/execution_policy.h>
-#include <thrust/sort.h>
-#include <thrust/extrema.h>
+struct QuantiledIndex {
+  size_t lower_bound;
+  size_t upper_bound;
+  size_t nearest;
+  double fraction;
+};
 
-#include <vector>
-#include <cassert>
-#include <algorithm>
-#include <functional>
-#include <rmm/thrust_rmm_allocator.h>
+QuantiledIndex find_quantile_index(gdf_size_type length, double quant);

```

Comment:
Does this need to be exposed? I'd think it'd only be used within `quantiles.cu`.

Reply:
Removed for now.
If gtest for `find_quantile_index` is required, I'd get it back.